src/components/atoms/interactiveAtom.tsx

```diff
@@ -5,7 +5,8 @@ import { Format } from '@guardian/types/Format';
 import { getPillarStyles, PillarStyles } from 'pillarStyles';
 import { Option, map, withDefault } from 'types/option';
 import { pipe2 } from 'lib';
-
+import {pageFonts} from "styles";
+import {remSpace} from "@guardian/src-foundations";
```

Comment:
For consistency?

```suggestion
import { pageFonts } from "styles";
import { remSpace } from "@guardian/src-foundations";
```

Reply:
This is intellij being it's weird and wonderful self